docs/index.md

```diff
@@ -4,54 +4,36 @@ title: Introduction
 nav_order: 1
 ---
 
-# Introduction
+<img src="/images/mainbanner.jpeg" alt="Friend Banner" width="100%">
 
-Friend is an AI wearable device that records everything you say, giving you proactive feedback and advice. Use these docs to unlock the full potential of Friend and build using the power of recall.
+# Introduction
 
-<img src="https://basedhardware.github.io/Friend/images/mainbanner.jpeg" alt="Friend Banner" width="100%">
+Meet Friend, the world’s leading open-source AI wearable that revolutionizes how you capture and manage conversations. Simply connect Friend to your mobile device and enjoy automatic, high-quality transcriptions of meetings, chats, and voice memos wherever you are.
 
 ## Features
 
-- Real-Time AI Audio Processing: Leverage powerful on-device AI capabilities for real-time audio analysis.
-
-- Low-powered Bluetooth: Capture audio for 24h+ on a small button battery
-
-- Open-Source Software: Access and contribute to the pin’s software stack, designed with openness and community collaboration in mind. -Wearable Design: Experience unparalleled convenience with ergonomic and lightweight design, perfect for everyday wear
-
-## How it works
-
-```mermaid
-graph TD;
-   A[Device] -- Streams Audio --> B[Phone App];
-   B -- Transmits --> C[Deepgram];
-   C -- Returns Transcript --> D[Phone App];
-   D -- Saves Transcript --> E[Phone Storage];
-
-classDef lightMode fill:#FFFFFF, stroke:#333333, color:#333333;
-classDef darkMode fill:#333333, stroke:#FFFFFF, color:#FFFFFF;
-
-classDef lightModeLinks stroke:#333333;
-classDef darkModeLinks stroke:#FFFFFF;
-
-class A,B,C,D,E lightMode
-class A,B,C,D,E darkMode
-
-linkStyle 0 stroke:#FF4136, stroke-width:2px
-linkStyle 1 stroke:#1ABC9C, stroke-width:2px
-linkStyle 2 stroke:#FFCC00, stroke-width:2px
-linkStyle 3 stroke:#2ECC40, stroke-width:2px
-```
+- Open-Source: Fully open-source, empowering you to customize and enhance your experience.
+- Long Battery Life: 24+ hour battery ensures you stay connected all day.
+- Live Transcription: Capture live voice and audio with human-level accuracy using OpenAI Whisper and Deepgram.
+- Conversation Memory: Remembers conversations, providing you with reliable records.
+- Efficient Summarization: Get instant insights and summaries powered by ChatGPT in just 5 seconds.
+- Connectivity: Bluetooth enabled with WiFi, LED, accelerometer, and USB-C charging.
```

Comment:
We don’t have Wifi, rest features are correct.

Reply:
Lol, I literally copied this from the description of our iOS app in App Store. :-)